x-pack/legacy/plugins/lens/public/app_plugin/app.tsx

```diff
@@ -239,7 +255,7 @@ export function App({
                 setState(s => ({ ...s, savedQuery }));
               }}
               onSavedQueryUpdated={savedQuery => {
-                data.query.filterManager.setFilters(savedQuery.attributes.filters || state.filters);
+                data.query.filterManager.setFilters(savedQuery.attributes.filters || []);
```

Comment:
Great as a fix for 7.6.1, but consider using https://github.com/elastic/kibana/pull/56160 for 7.7 to avoid similar issues :pray:

Reply:
The fix using that code is already in 7.7, but I couldn't backport it to 7.6